src/python/WMCore/WMRuntime/Monitors/PerformanceMonitor.py

```diff
@@ -59,7 +136,8 @@ def __init__(self):
 
         self.pid = None
         self.uid = os.getuid()
-        self.monitorBase = "ps -p %i -o pid,ppid,rss,pcpu,pmem,cmd -ww | grep %i"
+        self.monitorBase = "ps -p %i -o pid=,ppid=,rss=,pcpu=,pmem=,cmd= -ww | grep %i"
+        self.monitorForest = "ps --forest -o pid=,ppid=,rss=,pcpu=,pmem=,cmd= -g $(ps -o sid= -p %i)"
```

Comment:
This command actually returns an empty list and I cannot find what the problem is. It works just fine when executed in the terminal, or when interactively executed with python...